sirepo/pkcli/service.py

```diff
@@ -101,8 +101,7 @@ def jupyterhub():
     import sirepo.feature_config
 
     assert pkconfig.channel_in('dev')
-    assert 'jupyterhublogin' in sirepo.feature_config.cfg().sim_types, \
-        'you must set SIREPO_FEATURE_CONFIG_SIM_TYPES to contain the code "jupyterhublogin"'
+    assert 'jupyterhublogin' in sirepo.feature_config.cfg().other_sim_types
```

Comment:
I don't think this should check other_sim_types. It should use template.assert_sim_type.

Reply:
2b96cea4